client/src/App.jsx

```diff
@@ -0,0 +1,54 @@
+//Import styling sheets and functionalities
+import './App.css';
+import React, { useState, useContext } from 'react';
+
+
+//Import all components for use in display
+import ComponentContext from './ComponentContext.js';
+import NavBar from './components/NavBar.jsx';
+import HomePage from './components/HomePage.jsx';
+import About from './components/About.jsx';
+import Examples from './components/Examples.jsx';
+import Dashboard from './components/Dashboard.jsx';
+import AccountInformation from './components/AccountInformation.jsx';
+import SessionForm from './components/SessionForm.jsx';
+import HistoryList from './components/HistoryList.jsx';
+
+const App = () => {
+    //State to hold displayed component
+    const [component, setComponent] = useState('homepage');
+
+    //Function to switch between components - change to React Router!
```

Comment:
React Router is a better solution for handling navigation in React apps because it manages URLs, browser history, and dynamic routes. A switch statement is limited to simple conditional rendering. As your app grows, React Router is the better choice for more complex routing needs. I found this React router tutorial and hope it can be helpful: [React Router Tutorial](https://reactrouter.com/en/main/start/tutorial)

Reply:
Thank you, Xiahui! Refactoring to React Router will be a goal for the end of Week 2/start of Week 3. Thanks for the tutorial!